corejs and jasmine conflict

I'm using angular2 version 2.0.0 with angular2-highcharts 0.3.2
I'm getting these error on npm start:
`
node_modules/angular2-highcharts/typings/globals/es6-shim/index.d.ts(3,14): error TS2300: Duplicate identifier 'PropertyKey'.
typings/globals/core-js/index.d.ts(3,14): error TS2300: Duplicate identifier 'PropertyKey'.
typings/globals/jasmine/index.d.ts(74,9): error TS2375: Duplicate number index signature.
typings/globals/jasmine/index.d.ts(128,9): error TS2374: Duplicate string index signature.
`
Regards ,
Ali

Same issue here:
 node_modules/angular2-highcharts/typings/globals/es6-shim/index.d.ts(3,14): error TS2300: Duplicate identifier 'PropertyKey'. typings/globals/core-js/index.d.ts(3,14): error TS2300: Duplicate identifier 'PropertyKey'. typings/globals/jasmine/index.d.ts(74,9): error TS2375: Duplicate number index signature. typings/globals/jasmine/index.d.ts(128,9): error TS2374: Duplicate string index signature.

package.json content:
"scripts": {
    "build": "rm -rf dist && tsc",
    "test": "karma start karma.conf.js",
    "typings-install": "typings install",
    "watch": "ntsc -w"
  },

--- a/package.json
+++ b/package.json
@@ -5,8 +5,7 @@
   "scripts": {
     "watch": "ntsc -w",
     "build": "rm -rf dist && tsc",
-    "test": "karma start karma.conf.js",
-    "typings-install": "typings install"
+    "test": "karma start karma.conf.js"
   },
   "repository": {
     "type": "git",
@@ -34,6 +33,8 @@
     "@angular/platform-browser-dynamic": "^2.0.1",
     "@angular/platform-browser": "^2.0.1",
     "@angular/platform-server": "^2.0.1",
+    "@types/jasmine": "^2.2.34",
+    "@types/es6-shim": "^0.31.32",
     "core-js": "^2.4.1",
     "http-server": "^0.9.0",
     "jasmine-core": "~2.5.2",
@@ -48,6 +49,6 @@
     "zone.js": "0.6.25"
   },
   "dependencies": {
-    "highcharts": "^4.2.6"
+    "@types/highcharts": "^4.2.36"
   }
 }

--- a/src/Highcharts.ts
+++ b/src/Highcharts.ts
@@ -1,4 +1,4 @@
-/// <reference path="../typings/index.d.ts" />
+/// <reference types="highcharts" />
 declare var require: any;
 var Highcharts = require('highcharts/highstock.src');
 export { Highcharts };

--- a/typings.json
+++ b/typings.json
@@ -1,8 +1,0 @@
-{
-  "globalDependencies": {
-    "es6-shim": "registry:dt/es6-shim#0.31.2+20160602141504",
-    "highcharts": "registry:dt/highcharts#4.2.5+20160820095427",
-    "highcharts/highstock": "registry:dt/highcharts/highstock#2.1.5+20160316155526",
-    "jasmine": "registry:dt/jasmine#2.2.0+20160621224255"
-  }
-}

